scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -34,6 +34,15 @@ var _moth_jump_flutter_timer: float = 0.0
 var _moth_jump_initial_height: float = 0.0
 var _is_jump_button_held: bool = false
 
+## Used for rudimentary validation of exiting the climbable while holding up.
+## Change when tiles are implemented.
+var _is_up_pressed: bool = false  
+
+@export var LAUNCH_VELOCITY: int = 100
```

Comment:
While I am not a designer it seems very weird to have a jump be different on a ladder vs the ground. It just seems like it would create weird player expectations

Reply:
Removed launch velocity, players can choose to jump